### PR TITLE
[HE-401] Add Language selector

### PR DESCRIPTION
**Pull request by pedrogpimenta**
### Description
Add a Language selector on the top of the landing page and on the sidemenu when the user is not on the landing page.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-401

### CODE REVIEW

File: src/components/language-switcher/component.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const { page } = route;
+  const languages = useLanguages();
+  const locale = useLocale();
+
+  const renderLanguageSwitcher = () => languages.map(({ code }, index) => (
+    <button
+      className={
+        cx(
+          styles.languageButton,
+          locale === code && styles.languageButtonSelected,
```
Comment (Bluesmile82) [inline src/components/language-switcher/component.jsx:24]: If we don't have a language selected we should highlight 'en' as it is the default

**Review by Bluesmile82 — CHANGES_REQUESTED** [review]
Thanks for the work. Please check the comments

File: src/components/language-switcher/component.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      page === 'landing' && styles.alignCenter,
+    )}>
+      {renderLanguageSwitcher()}
+    </div>
+  )
+}
+
+// export default LanguageSwitcher;
+export default process.env.NODE_ENV === 'development'
+  ? hot(module)(connect(mapStateToProps, null)(LanguageSwitcher))
```
[the remark this answers is not in the document]
Comment (pedrogpimenta) [reply src/components/language-switcher/component.jsx:47]: We don't sorry!

**Review by Bluesmile82 — APPROVED** [review]
Great! Thank you for the changes

**Pull request merged**